packages/styled-components/src/components/primitives/Link/Link.styled.ts

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import { getTheme } from 'utils/getTheme';
+
+interface Props {
+  theme: any;
+}
+
+export const StyledLink = styled.a<Props>`
+  color: #000000;
```

Comment:
Can we change this color to `fillBlack500` color theme?

Reply:
done